playbooks/common/openshift-cluster/upgrades/pre/config.yml

```diff
@@ -64,7 +64,9 @@
   - fail: msg="Master running {{ openshift.common.version }} must be upgraded to {{ openshift_version }} before node upgrade can be run."
     when:
     - l_upgrade_nodes_only | default(False) | bool
-    - not openshift.common.version | match(openshift_version)
+    # we may get openshift_version as 3.x, 3.x.y, 3.x.y.z. Just run minor version check.
+    - openshift.common.version.split('.')[0:2] != openshift_version.split('.')[0:2]
```

Comment:
I don't think its correct, we're trying to maintain the same minor version for masters and nodes. In this case masters may run `3.9.2` and nodes would be `3.9.1` or `3.9.0`